howto/cluster/scale-down.md

```diff
@@ -1,24 +1,38 @@
 Scaling down a LXD cluster involves more checks than scaling up.
 
-First, you must pick a suitable candidate node. When a node is removed, all still running containers on it are stopped and potentially connected users are disconnected. To avoid this, AMS provides a feature to mark a node as unschedulable so that it will not be considered for any further container launches.
+There are two important things to know when scaling down:
+ - The node you remove **MUST NOT** have any container left.
+ - You **MUST** wait for a node to be fully removed before you can start removing another one
 
+1. Preparing the node for removal
```

Comment:
```suggestion
## Prepare the node for removal
```
Since these steps are quite long, let's turn them into sections. Not sure if we should keep the numbers - I'm okay with keeping them, but I don't think they are really needed. But up to you :)